### PR TITLE
Overhaul UniFi access points

### PR DESCRIPTION
**Pull request by nomaster**
Overhaul the Ubiquiti access point device types that currently exist in this repo, with the following goals:

* device names without 'UniFi'
* device part numbers according to Ubiquiti shop
* normalized slugs as manufacturer/part-number in lowercase, without market suffix (US/EU)
* device weight in grams (the Ubiquiti shop writes pounds in brackets)
* interface names according to operating system
* add missing details (PoE, air flow, etc.)
* comment with link to the entry in Ubiquiti store, if applicable
* air flow is always passive

### CODE REVIEW

**Comment by nomaster** [comment]
I'm glad if anyone is willing to proof-read the changes! 😃

**Comment by danner26** [comment]
Hello @nomaster per the repos guidelines, all slugs should have manufacturer appended to the slug. changing the slug from `ubiquiti-` to `unifi-` is incorrect. After you correct that, I would be happy to re-review. Ping me when ready, thanks!

**Comment by nomaster** [comment]
> Hello @nomaster per the repos guidelines, all slugs should have manufacturer appended to the slug. changing the slug from `ubiquiti-` to `unifi-` is incorrect.

Honestly, I think we were both wrong. The repo guideline say that the slug has to be unique per manufacturer, but they don't ask for a vendor name in them. Also, all other device types I checked in this repo didn't show such a thing. Consequently, I removed the 'unifi'.

> After you correct that, I would be happy to re-review. Ping me when ready, thanks!

Thanks for helping out! :)

**Comment by danner26** [comment]
The new format we are following is that the manufacturer needs to be appended to the slug. If it is not in the readme then I missed adding that and will take care of that. Sorry for the confusion. Either way, if the device is a `U6-Enterprise` then the slug should be `ubiquiti-u6-enterprise`

**Comment by danner26** [comment]
I have corrected the readme, thanks for letting me know

**Comment by nomaster** [comment]
I can't see what's wrong with the `yamlfmt` check. Anyone?

**Comment by danner26** [comment]
Hey @nomaster, any reason why you have changed the weight unit/value? The value already defined is the same as that in the datasheet. Please change these back.

**Comment by nomaster** [comment]
> Hey @nomaster, any reason why you have changed the weight unit/value? The value already defined is the same as that in the datasheet.

The datasheet and the website list the device weights in grams. An additional value is given in pounds, but written in brackets.

Also, temperature limits are given in degrees Celsius, with an additional value in Fahrenheit, but written in brackets.

This is why I think Ubiquiti prefers SI units and in my opinion we should too.

File: device-types/Ubiquiti/U6-Enterprise.yaml [hunk 1]
```diff
@@ -1,17 +1,23 @@
 ---
 manufacturer: Ubiquiti
-model: UniFi U6 Access Point Enterprise
+model: U6 Access Point Enterprise
```
Comment (danner26) [inline device-types/Ubiquiti/U6-Enterprise.yaml:3]: According to the datasheet this is a `Access Point U6 Enterprise`

File: device-types/Ubiquiti/U6-IW.yaml [hunk 2]
```diff
@@ -1,28 +1,43 @@
...
+    label: '1'
     type: 1000base-t
-  - name: Port 3
+    poe_mode: pse
+    poe_type: type2-ieee802.3at
+  - name: eth2
+    label: '2'
     type: 1000base-t
-  - name: Port 4
+    poe_mode: pse
```
Comment (danner26) [inline device-types/Ubiquiti/U6-IW.yaml:26]: The datasheet says there are 4 RJ45 ports, which ONE has POE output. Looking at the pictures, this is Port 1. Please remove the poe definitions on this interface

File: device-types/Ubiquiti/U6-IW.yaml [hunk 3]
```diff
@@ -1,28 +1,43 @@
...
+    poe_type: type2-ieee802.3at
+  - name: eth3
+    label: '3'
     type: 1000base-t
-  - name: Port 5
+    poe_mode: pse
```
Comment (danner26) [inline device-types/Ubiquiti/U6-IW.yaml:31]: The datasheet says there are 4 RJ45 ports, which ONE has POE output. Looking at the pictures, this is Port 1. Please remove the poe definitions on this interface

File: device-types/Ubiquiti/U6-IW.yaml [hunk 4]
```diff
@@ -1,28 +1,43 @@
...
+    poe_type: type2-ieee802.3at
+  - name: eth4
+    label: '4'
     type: 1000base-t
-    poe_mode: pd
+    poe_mode: pse
```
Comment (danner26) [inline device-types/Ubiquiti/U6-IW.yaml:36]: The datasheet says there are 4 RJ45 ports, which ONE has POE output. Looking at the pictures, this is Port 1. Please remove the poe definitions on this interface

File: device-types/Ubiquiti/U6-LR.yaml [hunk 5]
```diff
@@ -1,10 +1,21 @@
...
+weight: 800.0
+weight_unit: g
 interfaces:
-  - name: lan0
+  - name: eth0
     type: 1000base-t
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+  - name: wlan0
+    type: ieee802.11n
```
Comment (danner26) [inline device-types/Ubiquiti/U6-LR.yaml:19]: Where are you getting `n` from? This is an ax access point

File: device-types/Ubiquiti/U6-Lite.yaml [hunk 6]
```diff
@@ -1,10 +1,21 @@
...
+airflow: passive
+weight: 300.0
+weight_unit: g
 interfaces:
   - name: lan0
     type: 1000base-t
+    poe_mode: pd
+    poe_type: type1-ieee802.3af
+  - name: wlan0
+    type: ieee802.11n
```
Comment (danner26) [inline device-types/Ubiquiti/U6-Lite.yaml:19]: Where are you getting `n` from? This is an ax access point

File: device-types/Ubiquiti/U6-IW.yaml [hunk 7]
```diff
@@ -1,28 +1,43 @@
...
     poe_type: type2-ieee802.3at
-    mgmt_only: true
-  - name: Wlan 0
+  - name: wlan0
     type: ieee802.11n
-  - name: Wlan 1
+  - name: wlan1
     type: ieee802.11ac
```
Comment (danner26) [inline device-types/Ubiquiti/U6-IW.yaml:41]: Where are you getting `ac` from? This is an ax access point

File: device-types/Ubiquiti/U6-IW.yaml [hunk 8]
```diff
@@ -1,28 +1,43 @@
...
+  - name: eth4
+    label: '4'
     type: 1000base-t
-    poe_mode: pd
+    poe_mode: pse
     poe_type: type2-ieee802.3at
-    mgmt_only: true
-  - name: Wlan 0
+  - name: wlan0
     type: ieee802.11n
```
Comment (danner26) [inline device-types/Ubiquiti/U6-IW.yaml:39]: Where are you getting `n` from? This is an ax access point

File: device-types/Ubiquiti/U6-Mesh.yaml [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+airflow: passive
+weight: 400
+weight_unit: g
+interfaces:
+  - name: eth0
+    type: 1000base-t
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+  - name: wlan0
+    type: ieee802.11n
```
Comment (danner26) [inline device-types/Ubiquiti/U6-Mesh.yaml:19]: Where are you getting `n` from? This is an ax access point

**Review by danner26 — CHANGES_REQUESTED** [review]
Please review the current comments. Once you are done with those I will take another look

**Comment by danner26** [comment]
Hello, today we finished migrating all of the files and slugs to their new proper naming convention as described in the README. It looks like this PR has a merge conflict. I do not have access to modify this branch, so please resolve it by using the slug/model name defined in the master branch and then push it to this PR.

File: device-types/Ubiquiti/U6-Enterprise.yaml (hunk 1 above)
Comment (nomaster) [reply device-types/Ubiquiti/U6-Enterprise.yaml:3]:
```suggestion
model: Access Point U6 Enterprise
```

File: device-types/Ubiquiti/U6-IW.yaml (hunk 2 above)
Comment (nomaster) [reply device-types/Ubiquiti/U6-IW.yaml:26]: I'm not sure how we should document this. As I understand, one port (probably eth0, on the back side) is for the uplink and has PoE in. One of the output ports has PoE out, but I can't see if this is a fixed port or just any of them.

File: device-types/Ubiquiti/U6-LR.yaml (hunk 5 above)
Comment (nomaster) [reply device-types/Ubiquiti/U6-LR.yaml:19]:
```suggestion
    type: ieee802.11x
```

File: device-types/Ubiquiti/U6-Lite.yaml (hunk 6 above)
Comment (nomaster) [reply device-types/Ubiquiti/U6-Lite.yaml:19]:
```suggestion
    type: ieee802.11x
```

File: device-types/Ubiquiti/U6-Mesh.yaml (hunk 9 above)
Comment (nomaster) [reply device-types/Ubiquiti/U6-Mesh.yaml:19]:
```suggestion
    type: ieee802.11x
```

File: device-types/Ubiquiti/UAP-Pro.yaml [hunk 10]
```diff
@@ -1,10 +1,19 @@
...
+weight_unit: g
 interfaces:
   - name: eth0
     type: 1000base-t
+    poe_mode: pd
+    poe_type: type1-ieee802.3af
+  - name: wlan0
+    type: ieee802.11n
+  - name: wlan1
+    type: ieee802.11n
```
Comment (nomaster) [inline device-types/Ubiquiti/UAP-Pro.yaml:19]:
```suggestion
    type: ieee802.11ac
```

File: device-types/Ubiquiti/U6-Enterprise.yaml [hunk 11]
```diff
@@ -1,17 +1,23 @@
...
+  [U6 Access Point Enterprise](https://store.ui.com/products/u6-enterprise)
 interfaces:
-  - name: Port 1
+  - name: eth0
     type: 2.5gbase-t
     poe_mode: pd
     poe_type: type2-ieee802.3at
-  - name: Wlan 0
+  - name: wlan0
+    type: ieee802.11n
```
[the remark this answers is not in the document]
Comment (nomaster) [reply device-types/Ubiquiti/U6-Enterprise.yaml:19]:
```suggestion
    type: ieee802.11ax
```

File: device-types/Ubiquiti/U6-IW.yaml (hunk 7 above)
Comment (nomaster) [reply device-types/Ubiquiti/U6-IW.yaml:41]:
```suggestion
    type: ieee802.11ax
```

**Comment by nomaster** [comment]
What can I do?

File: device-types/Ubiquiti/U6-IW.yaml (hunk 2 above)
Comment (danner26) [reply device-types/Ubiquiti/U6-IW.yaml:26]: It is a fixed port. Specifically the first port on the left side of the device

**Review by danner26 — APPROVED** [review]

**Comment by nomaster** [comment]
Great, thanks @danner26! 🙏🏼 🌈

**Pull request merged**